components/FlexibleHeader/tests/snapshot/FlexibleHeaderViewAnimationDelegateTests.swift

```diff
@@ -16,15 +16,32 @@ import XCTest
 import MaterialComponents.MaterialFlexibleHeader
 
 private class MockMDCFlexibleHeaderViewAnimationDelegate: NSObject, MDCFlexibleHeaderViewAnimationDelegate {
-  var didCompleteExpectation: XCTestExpectation
-  init(didCompleteExpectation: XCTestExpectation) {
+  var didChangeAnimatedExpectation: XCTestExpectation?
+  var didChangeNotAnimatedExpectation: XCTestExpectation?
+  var didCompleteExpectation: XCTestExpectation?
```

Comment:
Nit: because these `XCTestExpectation`s are set in the constructor and never mutated, they can be made immutable (`var` -> `let`)

Reply:
Good catch! Fixed.